app/predacons_cli/src/cli.py

```
@@ -50,19 +62,42 @@ def launch(self,logs=False):
                                          config["auto_quantize"])
         
         chat  = []
+        vector_db = None
+        print("[yellow]Checking for data sources for the chat[/yellow]")
+        if config.get("scrap_web", False):
+            print("[yellow]Web scraping enabled iitializing web scraper[/yellow]")
+            try:
+                web_scraper = WebScraper()
+                print("[green]Web scraper initialized![/green]")
+            except:
```

Ruff

Do not use bare `except`

(E722)